typings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "globalDependencies": {
+    "mocha": "registry:dt/mocha#2.2.5+20161028141524",
```

Comment:
Yes, they don't have type definitions for the latest versions of those libraries. I'll update typings when new versions are available. 

`typings info dt~sinon --versions` -> version 1.16.1
`typings info dt~mocha --versions` -> version 2.2.5